src/ucp/api/ucp.h

```diff
@@ -2625,4 +2637,30 @@ ucs_status_t ucp_worker_flush(ucp_worker_h worker);
  */
 
 
+/**
+ * @ingroup UCP_COMM
+ * @brief Check the status of non-blocking request.
```

Comment:
Should it be marked as deprecated ?

Reply:
It's market https://github.com/openucx/ucx/pull/1777/files#diff-9b3b4a6fdccef100fb8608a63490414fR2658
i'll move it to ucp_compat.h